### ISSUE TITLE
Update or replace drf-jwt

### ISSUE THREAD

**Issue by Jawayria** [edx/upgrades#13]
We use the package [drf-jwt](https://github.com/Styria-Digital/django-rest-framework-jwt) in edx/completion, edx/course-discovery, edx/credentials, edx/demographics, edx/ecommerce, edx/edx-analytics-dashboard, edx/edx-analytics-data-api, edx/edx-drf-extensions, edx/edx-enterprise, edx/edx-enterprise-data, edx/edx-notes-api, edx/edx-organizations, edx/edx-platform, edx/edx-proctoring, edx/edx-rbac, edx/edx-val, edx/edx-when, edx/enterprise-catalog, edx/license-manager, edx/portal-designer, edx/registrar, edx/video-encode-manager. It hasn't yet added support for Django3.2. Please follow the guidance in https://openedx.atlassian.net/wiki/spaces/AC/pages/3036972032/Handling+Outdated+Dependencies to resolve the problem this poses for the Open edX Django 3.2 upgrade.

**Comment by GlugovGrGlib** [comment]
I have created an issue, asking for a support from a maintainer in  [drf-jwt](https://github.com/Styria-Digital/django-rest-framework-jwt) — https://github.com/Styria-Digital/django-rest-framework-jwt/issues/101
I'll keep you posted on it.

**Comment by natabene** [comment]
@GlugovGrGlib Thank you so much for picking this up, I appreciate! Please update us here on your progress.

**Comment by GlugovGrGlib** [comment]
@natabene Sorry for the delayed response, this ticket can be moved to review state and closed. The maintainer of the **drf-jwt** library has confirmed that there's no upper bound for Django dependency, and no code changes/release required.

We can continue using version [1.19.1](https://pypi.org/project/drf-jwt/1.19.1/), that was released May 18th 2021, in all edX repositories.

The only change since the released version that tox configuration was added, it allows testing the library against django3.2 env. I ran the tests against Python 3.8.12 and Django 3.2.8, all tests are passed, the total coverage is 95%.
